.github/ISSUE_TEMPLATE/staging-review.md

```diff
@@ -116,12 +117,15 @@ This may include mockups, prototypes, Github files, Word document or outline, st
 
 ## Scheduling
 
+- Open the [staging review calendar](https://calendly.com/collaboration-cycle/staging-review)
+- Select a date and time and click “Confirm”
+- Add your name and email
+- Click "Add Guests" and enter the VFS meeting attendees email addresses. 
+- Click Schedule Event
 <!-- 
-Please select your preferred time slot from the Platform Collaboration Meetings Calendar at
-https://calendar.google.com/calendar/u/0/embed?src=adhocteam.us_4dn3o77gcm5e3vbiedlha96tc0@group.calendar.google.com&ctz=America/New_York.
+Please note: You are responsible for adding the VFS atteendees. Platform will add the Platform reviewers. You will receive an email and the meeting will be automatically added to your calendar
```

Comment:
Maybe also say something about how if the list of attendees exceeded 10, they are able to forward the scheduled meeting to the others. This is what you had recommended we have as an FAQ.